### PR TITLE
chore(fsapp): upgrade react-native-device-info to 0.26.2

### PR DESCRIPTION
**Pull request by DavidRhodesBB**

### CODE REVIEW

**Comment by DavidRhodesBB** [comment]
https://github.com/rebeccahughes/react-native-device-info/blob/master/CHANGELOG.md

changes appear to be all additions to API, no breaking changes.

File: packages/pirateship/src/lib/analytics.ts [hunk 1]
```diff
@@ -9,7 +9,8 @@ import { CommerceTypes } from '@brandingbrand/fscommerce';
 
 import { Platform } from 'react-native';
 
-const DeviceInfo = require('react-native-device-info');
+// const DeviceInfo = require('react-native-device-info');
```
Comment (sdenaci) [inline packages/pirateship/src/lib/analytics.ts]: could you remove this console.log?

Comment (DavidRhodesBB) [reply packages/pirateship/src/lib/analytics.ts]: Sure thing! On it's way.

**Comment by Cauldrath** [comment]
This has been updated to ^0.29.1 elsewhere.

**Pull request closed without merging**